### ISSUE TITLE
Export loan parameters with parts

### ISSUE THREAD

**Issue by happiah-madson** [ArctosDB/arctos#8267]
# Tell us what you are trying to do 
When I am querying the collection for a user, I need to be able to easily determine the constraints of permits. These constraints largely fall into 4 categories:

1. no restrictions (just the standard terms of our MTA).
2. attribution for material usage.
3. notification and attribution for material usage.
4. extremely complex restrictions (recipient needs to have a permit to use the materials/CITES or ESA is implicated/it's a marine mammal/etc. etc. etc.)

I need to be able to see this information when I export parts to send someone who wants to loan materials so that we can ensure that the correct constraints are being written into our MTAs. It is not feasible for me to click into every Accession and review permits (every accession should have _at least 1_ documenting the transfer of the materials into the collection), especially the way they are set up right now (see: https://github.com/ArctosDB/dev/issues/98)

I've spoken to @mkoo and @campmlc about this and our need to make permits _functional_. If I can't interact with them in records and parts, I don't know why we would spend the time to enter them. @campmlc suggested entering a permit and then simultaneously entering an encumberance that annotates the consequences of the permit (must have attribution; no material transfer without prior approval, etc.) but that seems like double the work and no clear link between the permit and the encumbrance (which is public, and so seem real weird).

# What are relevant pages in Arctos
**Provide a link to or a description of the page where you need help.**

**Comment by dustymc** [comment]
>that seems like double the work


and a near-certainty of NOT doing both things from time to time - https://github.com/ArctosDB/arctos/issues/8241

>encumbrance (which is public

the existence and nature of, yes - not the details - still https://github.com/ArctosDB/arctos/issues/8241

I keep thinking I hear (latest right ^^ there!) 'we want use restrictions drawn from permits and summarized in various UI' and I've proposed the structure to do that (and publicly, but that's not strictly necessary, it would just possibly make things nicer for your users, all UI is details at this point) but it seems that's not quite right (or this issue wouldn't have been opened??) so I'm stuck.

I suspect we're not communicating effectively in this medium and a Committee (hoping a very short-lived one, but we could find out...) to find common ground is the best place to go from here.

**Comment by happiah-madson** [comment]
>I suspect we're not communicating effectively in this medium and a Committee (hoping a very short-lived one, but we >could find out...) to find common ground is the best place to go from here.

This makes sense to me! I don't know how to get that started, is there something I should do?

**Comment by happiah-madson** [comment]
(also 100% correct re: and a near-certainty of NOT doing both things from time to time)!!!!!

**Comment by dustymc** [comment]
>know how to get that started

I don't either, but please add it to https://github.com/ArctosDB/internal/issues/356 when you do! (I hadn't considered that aspect of things, but I think it should be part of the same hopeful-document.)

>100% correct re: and a near-certainty

I know! That's very nearly always involved in any technical objections I'm forced to lodge.

**Comment by mkoo** [comment]
This is on the agenda for Thursday

**Comment by dustymc** [comment]
I think this is resolved and implemented, tentatively closing.
